packages/strapi-plugin-content-manager/admin/src/containers/EditViewDataManagerProvider/utils/removePasswordFieldsFromData.js

```diff
@@ -3,14 +3,28 @@ import { get } from 'lodash';
 const getType = (schema, attrName) => get(schema, ['attributes', attrName, 'type'], '');
 const getOtherInfos = (schema, arr) => get(schema, ['attributes', ...arr], '');
 
-const removePasswordFieldsFromData = (data, contentTypeSchema, componentSchema) => {
+const removePasswordFieldsFromData = (data, contentTypeSchema, componentSchema, isClone) => {
```

Comment:
I really don't like the idea of mixing different purposes in the utility. Originally this helper is used to delete a specific type from entry, in this particular case the password type, in this way a possible improvement of this utility could be to allow to delete more attribute type by passing a argument to the function.

Although, the code works great for deleting other keys from an object I believe that your logic should be exported in a specific utility where we could define all the keys that needs to be removed from an object. 

Also it will be easier for you to add tests to it.